samples/solidity/test/ERC20WithData.ts

```diff
@@ -63,15 +63,6 @@ describe('ERC20WithData - Unit Tests', function () {
     expect(await deployedERC20WithData.balanceOf(signerB.address)).to.equal(0);
   });
 
-  it('Mint - Non-signing address should not be able to mint tokens', async function () {
```

Comment:
We lost a few tests here... I know this is just a backport, but might want to understand why they were lost